interpreter/interpreter.py

```diff
@@ -834,4 +838,15 @@ def messages_to_prompt(messages):
 
   def _print_welcome_message(self):
     current_version = pkg_resources.get_distribution("open-interpreter").version
-    print(f"\n Hello, Welcome to [bold white]⬤ Open Interpreter[/bold white]. (v{current_version})\n")
\ No newline at end of file
+    print(f"\n Hello, Welcome to [bold white]⬤ Open Interpreter[/bold white]. (v{current_version})\n")
```

Comment:
When processing this line, a conflict will occur. Please select the code of the main branch.